### DISCUSSION TITLE
TLS version settings in config

### DISCUSSION THREAD

**Discussion by m0zgen** [0xERR0R/blocky#531] [Q&A] [answered]
Hello, 

In the current version, `blocky` has support works with TLS through SSL configuration (key, cert, port) and on the current time as default can works on TLS 1.0, 1.1, 1.2, 1.3, will be good it it possible, if endpoint users will can customize TLS version in settings config file, as example:
```
httpsPort: 443
certFile: server.crt
keyFile: server.key
tls:
  - 1.2
  - 1.3
```
Where `tls` option enable working mode _only_ supporting on TLS v1.2, v1.3 

Thank you in advance for your attention.

**Comment by 0xERR0R** [comment, marked as answer]
With https://github.com/0xERR0R/blocky/commit/43d3e15549ed3aad333525095f1a6d4f5fca8ffe only 1.2 and 1.3 are supported now. Will be part of the next release.

**Reply by m0zgen** [reply]
🔥 thx
